### ISSUE TITLE
[Backport 2.8] Windows can't install apps in airgap setup

### ISSUE THREAD

**Issue by slickwarren** [rancher/rancher#44952]
backport of: https://github.com/rancher/rancher/issues/44949

**Comment by HarrisonWAffel** [comment]
This issue is highly likely to be a manifestation of SURE-8159, introduced by [this PR](https://github.com/rancher/rancher/pull/43160/files#diff-796d989f6c1367daebda4c3f2d5baed13e3ece991060dbcf049a535d9b609cf2L51-L58), which changed how charts are handled in air-gapped clusters. A PR is open to [revert](https://github.com/rancher/rancher/pull/45432) this change in 2.8-head. Once that PR is merged I will move this ticket into 'ready for qa' so that we can confirm that this is not a windows specific issue.

**Comment by HarrisonWAffel** [comment]
The PR has been merged into 2.8-head, so this issue is ready to be retested

**Comment by slickwarren** [comment]
tested on v2.8.6-rc1:
* provisioned a cluster in airgap setup using latest k8s version available (v1.28.11 +rke2r1)
* installed monitoring chart v103.1.1+up45.31.1  - pass
* installed latest version of logging - pass
